not sure if it's just my machine, or numpy:

```
$ nix build --no-link --keep-going --option build-use-sandbox relaxed -f /home/jon/.cache/nixpkgs-review/pr-96883/build.nix
builder for '/nix/store/flvaydgqqqlpjnhm3s4d1cxb444j3qv5-python3.8-cirq-0.8.2.drv' failed with exit code 1; last 10 log lines:
  test_example_runs_bcs_mean_field_perf                197,441.5890 (505.45)   246,864.2670 (541.90)   210,672.8178 (528.86)   20,686.5867 (>1000.0)  202,633.1730 (510.30)   19,923.2685 (>1000.0)       1;0      4.7467 (0.00)          5           1
  test_example_runs_hello_line_perf                    219,470.2300 (561.84)   309,193.5710 (678.72)   238,316.9818 (598.26)   39,638.7597 (>1000.0)  220,879.2550 (556.25)   24,512.7475 (>1000.0)       1;1      4.1961 (0.00)          5           1
  -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

  Legend:
    Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
    OPS: Operations Per Second, computed as 1 / Mean
  =========================== short test summary info ============================
  FAILED cirq/ion/convert_to_ion_gates_test.py::test_convert_to_ion_gates - ass...
  ===== 1 failed, 8644 passed, 36 xfailed, 255 warnings in 120.70s (0:02:00) =====
builder for '/nix/store/rn05sywqhwg835gq3g56mhs1cyh31lmg-python3.7-cirq-0.8.2.drv' failed with exit code 1; last 10 log lines:
  test_example_runs_bcs_mean_field_perf                211,374.1080 (555.23)   307,129.9690 (711.02)   239,041.2258 (614.29)   39,523.0382 (>1000.0)  226,088.3760 (582.58)   42,361.8932 (>1000.0)       1;0      4.1834 (0.00)          5           1
  test_example_runs_hello_line_perf                    252,885.7500 (664.27)   303,608.6790 (702.87)   272,028.4302 (699.06)   23,228.6949 (>1000.0)  257,157.8810 (662.64)   38,013.4590 (>1000.0)       1;0      3.6761 (0.00)          5           1
  -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

  Legend:
    Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
    OPS: Operations Per Second, computed as 1 / Mean
  =========================== short test summary info ============================
  FAILED cirq/ion/convert_to_ion_gates_test.py::test_convert_to_ion_gates - ass...
  ===== 1 failed, 8644 passed, 36 xfailed, 255 warnings in 131.97s (0:02:11) =====
cannot build derivation '/nix/store/3g24fz969li527ai4gfb550cc9fdhzbs-env.drv': 2 dependencies couldn't be built
[0 built (2 failed), 4 copied (45.0 MiB), 13.3 MiB DL]
error: build of '/nix/store/3g24fz969li527ai4gfb550cc9fdhzbs-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/96883
2 packages failed to build:
python37Packages.cirq python38Packages.cirq
```

--- a/pkgs/development/python-modules/cirq/default.nix
+++ b/pkgs/development/python-modules/cirq/default.nix
@@ -48,6 +48,15 @@ buildPythonPackage rec {
     })
   ];
 
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace "freezegun~=0.3.15" "freezegun" \
+      --replace "matplotlib~=3.0" "matplotlib" \
+      --replace "networkx~=2.4" "networkx" \
+      --replace "numpy~=1.16, < 1.19" "numpy" \
+      --replace "protobuf~=3.12.0" "protobuf"
+  '';
+
   propagatedBuildInputs = [
     freezegun
     google_api_core
@@ -64,7 +73,7 @@ buildPythonPackage rec {
   ];
 
   doCheck = true;
-  # pythonImportsCheck = [ "cirq" "cirq.Ciruit" ];  # cirq's importlib hook doesn't work here
+  # pythonImportsCheck = [ "cirq" "cirq.Circuit" ];  # cirq's importlib hook doesn't work here
   dontUseSetuptoolsCheck = true;
   checkInputs = [
     pytestCheckHook
@@ -78,16 +87,10 @@ buildPythonPackage rec {
 
   pytestFlagsArray = [
     "--ignore=dev_tools"  # Only needed when developing new code, which is out-of-scope
+    "--benchmark-disable" # Don't need to run benchmarks when packaging.
   ];
   disabledTests = [
-    "test_serialize_sympy_constants"  # fails due to small error in pi (~10e-7)
-    "test_convert_to_ion_gates" # fails due to rounding error, 0.75 != 0.750...2
-
-    # Newly disabled tests on cirq 0.8
-    # TODO: test & figure out why failing
-    "engine_job_test"
-    "test_health"
-    "test_run_delegation"
+    "test_convert_to_ion_gates" # fails on some systems due to rounding error, 0.75 != 0.750...2
   ] ++ lib.optionals stdenv.isAarch64 [
     # Seem to fail due to math issues on aarch64?
     "expectation_from_wavefunction"
@@ -97,6 +100,7 @@ buildPythonPackage rec {
   meta = with lib; {
     description = "A framework for creating, editing, and invoking Noisy Intermediate Scale Quantum (NISQ) circuits.";
     homepage = "https://github.com/quantumlib/cirq";
+    changelog = "https://github.com/quantumlib/Cirq/releases/tag/v${version}";
     license = licenses.asl20;
     maintainers = with maintainers; [ drewrisinger ];
   };

